src/go/k8s/kuttl-v2-test.yaml

```diff
@@ -17,7 +17,9 @@ commands:
   - command: helm repo update
   - command: helm install --set logLevel=trace --set image.tag=dev
       --set image.repository=localhost/redpanda-operator --namespace redpanda --create-namespace
-      redpanda-operator redpanda/operator
+      redpanda-operator redpanda/operator --set additionalCmdFlags="{--additional-controllers=all}"
+  - command: "sh -c 'until kustomize build config/e2e-v2-tests 2>> tests/_e2e_artifacts/kustomize-output.txt | kubectl apply -f - 1>> tests/_e2e_artifacts/kubectl-output.txt 2>> tests/_e2e_artifacts/kubectl-error-output.txt; do sleep 0.5; done'"
+    background: true
```

Comment:
This can not work. The helm install could overwrite this kustomize build if it is set to background.

Reply:
Only we we upgrade you mean? I do not understand "background" Since the tests do not upgrade the operator, this should not happen. I have also tested this locally.